label_studio/tasks/models.py

```diff
@@ -976,6 +984,13 @@ def save(self, *args, **kwargs):
             logger.warning('project_id is not set for prediction, project_id being set in save method')
             self.project_id = Task.objects.only('project_id').get(pk=self.task_id).project_id
 
+        # make sure the source of this prediction is set correctly
+        if self.model is not None and self.model_run is not None:
+            logger.error(
+                'Only one of model or model_run should be set in prediction, '
+                'it cannot come from both an MLBackend and a ModelRun.',
+            )
+
```

Comment:
Not sure this will always be true, but seems reasonable to me for now.

Reply:
Why would this be true? model and model run are different entities right?

Reply:
Yeah I dont think it will ever happen, but maybe a good check to keep because we might accidentally set `model` instead of `model_version` :D

Reply:
will have to figure out what to do with this now that the previous change is reverted in develop

Reply:
reverted this check until HUMSIG-37 gets remerged